newIDE/app/src/Profile/Subscription/SubscriptionDetails.js

```diff
@@ -81,12 +84,14 @@ const subscriptionOptions: {
 
 type Props = {
   subscription: ?Subscription,
+  subscriptionPlansWithPrices: SubscriptionPlanWithPrices[],
   onManageSubscription: () => void | Promise<void>,
   isManageSubscriptionLoading: boolean,
   simulateNativeMobileApp?: boolean,
 };
 
 /**
+ * TODO: Update doc
```

Comment:
todo?

Reply:
I thought something would change but it didn't in the end